src/modules/Authentication/components/FormField/index.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+
+import Form from 'react-bootstrap/Form'
+
+export default (props) => {
+  const { type = 'text', name, required = true, invalidFeedback = defaultInvalidFeedbackGenerator } = props
```

Comment:
React provides a way to validate props. Take a look at this https://reactjs.org/docs/typechecking-with-proptypes.html.

Reply:
Do I have to validate all props or is it because of the next line?
```js
const invalidFeedbackString = typeof invalidFeedback === 'function' ? invalidFeedback(props) : invalidFeedback
```
If it's because of that line, I would have to check if it's a function anyway. The idea is that you can pass a function or a string as the `invalidFeedback` prop and if it's a function, the component calls it to get the `invalidFeedbackString`.

Reply:
Not 100% sure if there is any "best practice" about this but if I were you I'd validate all those props that could "break" your component.

And about your validation, you can do that with React. My suggestion is to validate first about the props' types and then to do that "coalesce" with them. Right now, I could send an Array as name and it would crash, wouldn't it?

Reply:
Yes, you're right. I'll add typechecking and replace form logic with Formik. Thanks!